components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/AddonsManagerAdapter.kt

```diff
@@ -327,7 +333,9 @@ class AddonsManagerAdapter(
         val addonSummaryTextColor: Int? = null,
         val sectionsTypeFace: Typeface? = null,
         @ColorRes
-        val addonBackgroundIconColor: Int? = null
+        val addonBackgroundIconColor: Int? = null,
+        @DrawableRes
+        val addonAllowPrivateBrowsingLabelDrawableRes: Int? = null
```

Comment:
Should we also expose a colour to change the tint of the icon, as in Fenix icons tint can change depending on the theme colour ( dark, light)? I'm not sure if this icon should adapt to theme changes

Reply:
Hello @Amejia481, I exposed directly the icon (addonAllowPrivateBrowsingLabelDrawableRes) with the colors changed depending on the theme ([here](https://github.com/mozilla-mobile/fenix/blob/b3b0fdf1e13ca9dcebe730b44aab5290cff40a02/app/src/main/res/drawable/ic_add_on_private_browsing_label.xml#L1) is the svg from Fenix where I used addOnPrivateBrowsingExteriorCircleBackground and addOnPrivateBrowsingInteriorIconBackground attributes to change the colors depending on the theme. After that, I pass this resource in AddonsManagerAdapter.Style constructor. You can check it [here](https://github.com/mozilla-mobile/fenix/blob/b3b0fdf1e13ca9dcebe730b44aab5290cff40a02/app/src/main/java/org/mozilla/fenix/addons/AddonsManagementFragment.kt#L131).)

Reply:
If colour is changes when switching themes it's OK  :)